### PR TITLE
Install KMM from the bundle

### PR DESCRIPTION
**Pull request by qbarrand**
Skip `make deploy` and install KMM from the bundle instead to leverage image replacements.

### CODE REVIEW

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 base_images:
+  go-builder:
```
Comment (ybettan) [inline ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml:2]: We want all those changes in the relelase-1.0 branch as well I guess?

**Comment by qbarrand** [comment]
/pj-rehearse ci/rehearse/rh-ecosystem-edge/kernel-module-management/main/check-api-changes

**Comment by qbarrand** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-kernel-module-management-main-check-api-changes

**Comment by qbarrand** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-kernel-module-management-main-check-api-changes

**Review by enriquebelarte — APPROVED** [review]

**Comment by qbarrand** [comment]
/override pull-ci-rh-ecosystem-edge-kernel-module-management-main-check-commits-count
/override pull-ci-rh-ecosystem-edge-kernel-module-management-release-1.0-check-commits-count

File: ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml (hunk 1 above)
Comment (qbarrand) [reply ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml:2]: We do indeed. Added them to `release-1.0`.

**Pull request merged**